data_adapter_oemof/adapters.py

```diff
@@ -313,7 +328,21 @@ def parametrize_dataclass(
         cls, data: dict, timeseries: pandas.DataFrame, struct: dict
     ):
         defaults = cls.get_default_parameters(data, timeseries, struct)
-        defaults.update({"type": "ExtractionTurbine"})
+        for key, value in {"type": "ExtractionTurbine",
+                         "carrier": "carrier",
+                         "tech": "tech",
+                         "electricity_bus": "electricity_bus",
+                         "heat_bus": "heat_bus",
+                         "fuel_bus": "fuel_bus",
+                         "condensing_efficiency": 1,
+                         "electric_efficiency": 1,
+                         "thermal_efficiency": 1,
+                         }:
```

Comment:
I disagree with adding all those default kwargs.
The whole point of the Adapter class was to have as minimal effort to add new Adapters.
But instead to compare give data with related Facade and try to match it automatically, maybe make some smart guesses about the busses.
Having this implementation, we have lots of extra work (we don't have time for)!